test/node/download-webseed-torrent.js

```diff
@@ -9,8 +9,12 @@ var serveStatic = require('serve-static')
 var test = require('tape')
 var WebTorrent = require('../../')
 
+// it should be fast to download a small torrent over local HTTP
+var WEB_SEED_TIMEOUT_MS = 500
+
 test('Download using webseed (via .torrent file)', function (t) {
   t.plan(6)
+  t.timeoutAfter(WEB_SEED_TIMEOUT_MS)
```

Comment:
It completes very fast (~10ms) on my Macbook.

I added the timeout to match the timeout below. How do you verify that `webSeeds: false` is working? I thought: make sure that the webSeed download always completes in under X time, then set `webSeeds: false` and wait that same amount of time and verify there are no requests